src/reconstruct/reconstruct.ts

```diff
@@ -114,7 +114,7 @@ function reconstructUnaryOp(leaf: RNodeWithParent, operand: Code, configuration:
 */
 function reconstructBinaryOp(n: RBinaryOp<ParentInformation> | RPipe<ParentInformation>, lhs: Code, rhs: Code, configuration: ReconstructionConfiguration): Code {
 	if(isSelected(configuration, n)) {
-		return plain(getLexeme(n))
+		return plain(getLexeme(n),n.location? n.location.start.column : 0)
```

Comment:
for everything but expression-lists you should be guaranteed to receive a location, so these checks are only required if the respective node _could_ be an `RExpressionList`.